app/frontend/src/styles/application.scss

```diff
@@ -40,6 +40,7 @@ $govuk-image-url-function: frontend-image-url;
 @import 'application/tabs';
 @import 'application/timeline';
 
+@import 'application/jobSeekers/account';
```

Comment:
For consistency, can we have jobseekers as one word please?

Reply:
There's a whole existing folder here already – agree we should do a wholesale rename of these but as part of this PR I wanted to keep it consistent.